editor/editor_path.cpp

```diff
@@ -116,14 +117,22 @@ void EditorPath::update_path() {
 			continue;
 		}
 
-		Ref<Texture2D> icon = EditorNode::get_singleton()->get_object_icon(obj);
+		Ref<Texture2D> icon;
+		if (Object::cast_to<MultiNodeEdit>(obj)) {
+			icon = EditorNode::get_singleton()->get_class_icon(Object::cast_to<MultiNodeEdit>(obj)->get_edited_class_name());
```

Comment:
```suggestion
		if (MultiNodeEdit *mne = Object::cast_to<MultiNodeEdit>(obj)) {
			icon = EditorNode::get_singleton()->get_class_icon(mne->get_edited_class_name());
```
Same.

Reply:
We don't allow using assignments in `if` conditions.